src/unstructured_client/models/shared/partition_parameters.py

```diff
@@ -124,6 +124,10 @@ class PartitionParametersTypedDict(TypedDict):
     r"""This parameter determines if the PDF file should be split on the client side. It's an internal parameter for the Python client and is not sent to the backend."""
     split_pdf_page_range: NotRequired[List[int]]
     r"""When `split_pdf_page is set to `True`, this parameter selects a subset of the pdf to send to the API. The parameter is a list of 2 integers within the range [1, length_of_pdf]. A ValueError is thrown if the given range is invalid. It's an internal parameter for the Python client and is not sent to the backend."""
+    split_pdf_cache_tmp_data: NotRequired[bool]
```

Comment:
we can't edit these files manually as they will be removed the next time we sync from speakeasy

Reply:
I must have mixed up the files where we define new custom params - on it.